### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Brandonbr1**
Link email   stopped duplicate's and false repost site and improve folder structure

### CODE REVIEW

**Comment by Brandonbr1** [comment]
note: Allow edits and access to secrets by maintainers is enabled

File: README.md
Comment (dada513) [inline README.md:9]: Pretty sure [at] was to prevent some bots from spamming the email.

File: README.md
Comment (dada513) [inline README.md:28]:
```suggestion

```

Not needed, browser plugin is mentioned above.

File: lists/SITES.md
Comment (dada513) [inline lists/SITES.md:5]: Please rename this back to SITES.md, this file is linked by a couple of sites

File: scripts/generator.py
Comment (dada513) [inline scripts/generator.py:1]: As for this rename to scripts folder, I think it's cleaner, but would require reconfiguration of cloudflare pages before merge.

**Comment by Brandonbr1** [comment]
@dada513  did you changes you asked me to do😊😁

**Comment by Brandonbr1** [comment]
pls git squash this commit😊

File: README.md
Comment (dada513) [inline README.md:11]: why not `StopModReposts`?

File: README.md
Comment (dada513) [inline README.md:21]: ditto ^

File: README.md
Comment (dada513) [inline README.md:24]:
```suggestion
To report a site which is illegally redistributing mods, please open an issue in our [issue tracker](http://git.io/jaB7), submit a pull request or join our [Discord sever](https://discord.gg/zg4tt2M) and use the `/submit` command in the `#submit-site` channel. When submitting a site, please make sure it is not already on our [list](https://stopmodreposts.org/sites) and a "repost site" that links backs to the original download page does not count as a repost site. We are currently targeting reposting of the following content types: mods, resource packs, maps, and modpacks. In addition, we target websites providing malicious Minecraft content.
```

File: README.md
Comment (dada513) [inline README.md:28]:
```suggestion

```
I feel like this doesn't have to be mentioned. CODE_OF_CONDUCT.md is visible in the repo so anyone who wants will read it and github seems to often link it aswell when contributing

File: README.md
Comment (dada513) [inline README.md:40]:
```suggestion

```

This is fine but we are still reviewing the licensing terms so please remove for now until we properly do this

**Review by dada513 — CHANGES_REQUESTED** [review]
other than this all good

**Comment by dada513** [comment]
> pls git squash this commitblush

seems that squash is disabled, anyways you can git rebase it and force push your branch to do it manually

File: README.md
Comment (Brandonbr1) [reply README.md]: > Pretty sure [at] was to prevent some bots from spamming the email.

you could just use an [emailencoder](http://www.wbwip.com/wbw/emailencoder.html) and that should stop the bots

Comment (berrysauce) [reply README.md]: Using [at] is like dada said to prevent common email spam while still preserving readability compared to an encoded email address

**Pull request closed without merging**
